docs/src/Checkbox.doc.js

```diff
@@ -287,6 +287,7 @@ card(
     description="Here is an example of a checkbox showing an error message."
     id="hasError"
     name="Example: Error state"
+    skipContrastCheck
```

Comment:
This is a fix available until the docs are converted to the new doc components - therefore, not documenting to avoid it being abused in the short term